src/views/Events/Events.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import {
```

Comment:
If I understand the structure correctory, everything in 'views/' corresponds to a rounter switch. If thats the case I think these should just be wrappers. Ie, import the container component that you actually want to render, and then render it. 

```suggestion
import EventsDashboard from '../containers/EventsDashboard'
export default () => (
 <EventsDashboard />
)
```

Reply:
also I would name this file index.js